tests/unit/datarobot_drum/model_metadata/test_model_metadata.py

```diff
@@ -208,8 +208,10 @@ def yaml_str_to_schema_dict(yaml_str: str) -> dict:
         return schema.data
 
     def test_default_typeschema(self):
-        validator = SchemaValidator(type_schema={}, use_default_type_schema=True)
-        assert validator._using_default_type_schema
+        type_schema = read_model_metadata_yaml(SchemaValidator.DEFAULT_TYPE_SCHEMA_CODEDIR_PATH)[
```

Comment:
what is the purpose of this test?  does it test that the default schema is as expected?

Reply:
looks like that